server/game/cards/02_SHD/events/UnexpectedEscape.ts

```diff
@@ -0,0 +1,35 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { RelativePlayer, WildcardZoneName, ZoneName } from '../../../core/Constants';
+
+export default class UnexpectedEscape extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '1973545191',
+            internalName: 'unexpected-escape',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Exhaust a unit. You may rescue a captured card guarded by that unit.',
+            targetResolvers: {
+                exhaust: {
+                    zoneFilter: WildcardZoneName.AnyArena,
+                    controller: RelativePlayer.Opponent,
```

Comment:
I guess technically it could be a unit you control, it doesn't say "an enemy unit".